### PR TITLE
adding index page of Ecom website

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

**Comment by malaykhakhar** [comment]
While scrolling on homepage, the main container is getting over the navbar, and the navbar is visible below it. Rectify this for better user experience.

**Review by malaykhakhar — CHANGES_REQUESTED** [review]
While scrolling on homepage, the main container is getting over the navbar, and the navbar is visible below it. Rectify this for better user experience.

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="login.css">
+</head>
+<body>
+    <!----
+    <button onclick="document.getElementById('id01').style.display='block'" style="width:auto;">Login</button>
```
Comment (malaykhakhar) [inline login/login.html]: Remove comments which are not useful

File: login/login.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <input type="text" placeholder="Enter Username" name="uname" required>
+    
+          <label for="psw"><b>Password</b></label>
+          <input type="password" placeholder="Enter Password" name="psw" required>
+            
+          <button class="submit-btn">Login</button>
+          <label>
+              <!----
+            <input type="checkbox" checked="checked" name="remember"> Remember me
+          </label>
```
Comment (malaykhakhar) [inline login/login.html]: Again, same issue with comments

**Review by malaykhakhar — CHANGES_REQUESTED** [review]

File: clhome.css [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    width: 30%;
+    border-radius: 2rem;
+    cursor: pointer;
+    position: absolute;
+    left: 12rem;
+    bottom: 14rem;
+    text-decoration: none;
+}
+.content a:hover{
+    border:1px solid grey;
```
Comment (sri-rishi) [inline clhome.css]: use variable color naming

File: clhome.css [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+    padding:0.2rem;
+    margin-top:0.5rem;
+    color:white;
+    font-weight: 500;
+}
+.footer-content-love{
+    text-align:center;
+    padding-bottom:0.2rem;
+    margin:0.5rem;
+    color:white;
```
Comment (sri-rishi) [inline clhome.css]: here also use variable color naming

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Remove all comments form file

**Review by malaykhakhar — APPROVED** [review]
Looks good

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**